src/libs/actions/User.ts

```diff
@@ -982,8 +986,31 @@ function clearUserErrorMessage() {
     Onyx.merge(ONYXKEYS.USER, {error: ''});
 }
 
-function setMuteAllSounds(isMutedAllSounds: boolean) {
-    Onyx.merge(ONYXKEYS.USER, {isMutedAllSounds});
+function togglePlatformMute(platform: Platform, mutedPlatforms: Platform[]) {
+    const isPlatformMuted = mutedPlatforms?.includes(platform);
+    const newMutedPlatforms = isPlatformMuted ? mutedPlatforms.filter((mutedPlatform) => mutedPlatform !== platform) : [...mutedPlatforms, platform];
```

Comment:
Ah, sorry. I guess this was something I didn't explicitly say. It's actually an object like this:

```
{
    'web': true,
    'desktop': true,
}
```

and if a platform is muted, it is removed from the object entirely.

Reply:
Done!